src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(x) {
+      calculator.result += x;
+    },
+
+    subtract(x) {
+      calculator.result -= x;
+    },
+
+    multiply(x) {
+      calculator.result *= x;
+    },
+
+    divide(x) {
+      calculator.result /= x;
+    },
+
+    reset() {
+      calculator.result = 0;
+
+      return this;
```

Comment:
The `reset` method should return `calculator` for consistency with the `operate` method, which returns `this`. Since `this` refers to the `calculator` object in this context, it's more consistent to use `this` instead of `calculator`.